providers/common/include/prov/rand_local.h

```diff
@@ -0,0 +1,360 @@
+/*
```

Comment:
Why do you name this file "rand_*local*.h"?

Reply:
Because everything is mixed up in my head still isn't a good answer :)
I'll rename it.

Reply:
It's old residue too.  The equivalent is in implementations/rand.